tests/functional/test_install_extras.py

```diff
@@ -27,13 +27,13 @@ def test_extras_after_wheel(script, data):
         'install', '--no-index', '-f', data.find_links,
         'requires_simple_extra', expect_stderr=True,
     )
-    assert simple not in no_extra.files_created, no_extra.files_created
+    no_extra.did_not_create(simple)
```

Comment:
```suggestion
    no_extra.did_not_create(simple, no_extra.files_created)
```

Reply:
No -- this information is included in `str(result)`.